pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -310,22 +309,22 @@ func (a Adapter) Deploy(parameters common.DeployParameters) (err error) {
 				}
 			}
 
-			s := log.Spinnerf("Deploying the manifest for %s", gvk.Kind)
+			actionType := "Creating"
+			if instanceFound {
+				actionType = "Updating" // Update deployment
+			}
+			s := log.Spinnerf("%s resource of kind %s", strings.Title(actionType), gvk.Kind)
 			result := &unstructured.Unstructured{}
-			actionType := "create"
 			if !instanceFound {
 				result, err = a.Client.DynamicClient.Resource(gvr).Namespace(namespace).Create(deploymentManifest, metav1.CreateOptions{})
 			} else {
-				actionType = "update" // Update deployment
 				result, err = a.Client.DynamicClient.Resource(gvr).Namespace(namespace).Update(deploymentManifest, metav1.UpdateOptions{})
 			}
 			if err != nil {
 				s.End(false)
 				return errors.Wrapf(err, "Failed to %s manifest %s", actionType, gvk.Kind)
```

Comment:
This will now produce a weird error:  "Failed to Updating manifest ResourceComponent", due to the actionType being either "Creating" or "Updating", whereas it should be something like " Failed to update manifest ..."

Reply:
will just change `to` to `when`